### PR TITLE
[DBIO-1350] fix: lab cannot change address if status Verified

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA Link
[https://blocksphere2020.atlassian.net/browse/DBIO-1350](url)

### Changelog / Description
-add condition if account lab status verified. lab only change profile picture, email, phone number and web address

### CODE REVIEW

File: src/views/Dashboard/Lab/Account/index.vue [hunk 1]
```diff
@@ -361,6 +364,18 @@ export default {
         this.imageUrl = ""
       }
     },
+
+    editInfo() {
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/Account/index.vue:368]: What is this function for? Nothing calls this function.

File: src/views/Dashboard/Lab/Account/index.vue [hunk 2]
```diff
@@ -361,6 +364,18 @@ export default {
...
+      if (this.isEditable) {
+        this.isEditable = true
+      } else {
+        return this.isEditable = false
+      }
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/Account/index.vue:373]: The `if` statements are weird what are you trying to achieve?

File: src/views/Dashboard/Lab/Account/index.vue [hunk 3]
```diff
@@ -52,7 +52,7 @@
                   placeholder="Lab Name"
                   autocomplete="off"
                   outlined
-                  :disabled="!isEditable"
+                  :disabled="!isEditable || !isVerify"
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/Account/index.vue:55]: The issue only restricts the changing of Country, State, City, and Address. Not lab names.

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/Account/index.vue (hunk 3 above)
Comment (agustinustheo) [reply src/views/Dashboard/Lab/Account/index.vue:55]: Is there a mistake in the issue? Please confirm with @hilyds on this issue.

File: src/views/Dashboard/Lab/Account/index.vue (hunk 2 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/Account/index.vue:373]: if status account is verified, lab cannot change(edit) addres, city, and region.

File: src/views/Dashboard/Lab/Account/index.vue (hunk 3 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/Account/index.vue:55]: from the last time discussion with @hilyds , if lab status is verified, lab only can change email, phone number, website address and profile picture.

File: src/views/Dashboard/Lab/Account/index.vue (hunk 2 above)
Comment (agustinustheo) [reply src/views/Dashboard/Lab/Account/index.vue:373]: I still don't get it, please change the logic, it's very confusing. Why don't you just return the `this.isEditable` attribute instead?

**Review by agustinustheo — CHANGES_REQUESTED** [review]
Change in logic, and code clean up please.

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**